pkg/devfile/header.go

```diff
@@ -43,6 +43,10 @@ type DevfileMetadata struct {
 	// +optional
 	Tags []string `json:"tags,omitempty"`
 
+	// Optional devfile architecture, amd64 is considered the default architecture unless specified otherwise
```

Comment:
to be clear, it just declares which architectures are supported by all images used in devfile?
Is it supposed to be filled by devfile maintainer or we have a plan for automated evaluation of it during devfile registry build?

Reply:
The devfile maintainer could fill it out, and registry build could either validate that or provide/restrict based on content of the devfile like the architectures of any containers used.

Reply:
I've updated the post in the issue https://github.com/devfile/api/issues/244#issuecomment-852157289 with what is the expected behaviour with the arch field during registry build

PTAL